service.toml

```diff
@@ -2,11 +2,41 @@ name = "minio"
 
 [namespace.storage]
 
+[namespace.storage.new]
+required = ["endpoint", "name", "credential"]
+optional = ["work_dir", "use_ssl", "region"]
+
 [namespace.storage.op.list]
-optional = ["list_mode"]
+optional = ["list_mode", "continuation_token", "list_buffer_size"]
 
 [namespace.storage.op.read]
-optional = ["offset", "io_callback", "size"]
+optional = ["offset", "io_callback", "size", "file_version"]
```

Comment:
We intend to add a global pair to control specify the object version, let's ignore it for now.